pkg/kv/kvclient/kvcoord/dist_sender.go

```diff
@@ -249,7 +259,9 @@ func makeDistSenderMetrics() DistSenderMetrics {
 		SlowRPCs:                metric.NewGauge(metaDistSenderSlowRPCs),
 		RangefeedRanges:         metric.NewGauge(metaDistSenderRangefeedTotalRanges),
 		RangefeedCatchupRanges:  metric.NewGauge(metaDistSenderRangefeedCatchupRanges),
-		RangefeedErrorCatchup:   metric.NewCounter(metaDistSenderRangefeedErrorCatchupRanges),
+		RangefeedCatchupDuration: metric.NewHistogram(metaDistSenderRangefeedCatchupDuration,
+			base.DefaultHistogramWindowInterval(), (30 * time.Second).Nanoseconds(), 1),
```

Comment:
Is 30 seconds a suitable max here? I don't have a good sense of the expected latency in production clusters, but seems like this could easily run for a few minutes?